core/src/main/scala/knobs/Parser.scala

```diff
@@ -98,7 +98,6 @@ object ConfigParser {
 
   lazy val ident: Parser[Name] = for {
     n <- satisfy(c => Character.isLetter(c)).map2(takeWhile(isCont))(_ +: _)
-    _ <- failWhen[Parser](n == "import", s"reserved word ($n) used as an identifier")
```

Comment:
I deleted this line because it doesn't do anything. It's a comparison between an `Array` and a `String` so it will never match. Fixing the equality problem is not very difficult (and making it typesafe), but after writing a test for this and trying to get the error to bubble up to the top, I gave up.

In any event, the parser hasn't been restricting this so far and it seems to work...